pkg/podvolume/backupper.go

```diff
@@ -201,6 +202,27 @@ func (b *backupper) BackupPodVolumes(backup *velerov1api.Backup, pod *corev1api.
 			continue
 		}
 
+		if resPolicies != nil {
```

Comment:
Same suggestion: Let's wrap these checking code into a function, which helps reduce the complexity of the main workflow

Reply:
updated